src/unistd.rs

```diff
@@ -601,6 +672,69 @@ pub fn lseek64(fd: RawFd, offset: libc::off64_t, whence: Whence) -> Result<libc:
     Errno::result(res).map(|r| r as libc::off64_t)
 }
 
+/// call the link function to create a link to a file
```

Comment:
Please capitalize the first word and add a period to the end of the first line of all these doc comments. If you run `cargo doc --no-deps --open` you can easily review their appearance to make sure things look good there.